Web Inspector: Screenshots timeline overview records do not adjust their position when zooming in/out

patrickangle opened the pull request:
#### 554e2e83586998432ddbbb1bb7b7182c63960697
<pre>
Web Inspector: Screenshots timeline overview records do not adjust their position when zooming in/out
<a href="https://bugs.webkit.org/show_bug.cgi?id=242341">https://bugs.webkit.org/show_bug.cgi?id=242341</a>
rdar://96325691

Reviewed by Devin Rousso.

The position of the images should not only be set during initialization of the record&apos;s image element, but also during
each layout.

* Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineOverviewGraph.js:
(WI.ScreenshotsTimelineOverviewGraph.prototype.layout):

Canonical link: <a href="https://commits.webkit.org/252148@main">https://commits.webkit.org/252148@main</a>
</pre>

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineOverviewGraph.js:
```diff
@@ -67,10 +67,9 @@ WI.ScreenshotsTimelineOverviewGraph = class ScreenshotsTimelineOverviewGraph ext
-            this.element.appendChild(this._imageElementForRecord.getOrInitialize(record, () => {
+            let recordElement = this.element.appendChild(this._imageElementForRecord.getOrInitialize(record, () => {
                 let imageElement = document.createElement("img");
                 imageElement.height = this.height;
```
NIT: I realize that `this.height` is a constant, but maybe we should move this outside too in case that changes in the future 🤔

patrickangle replied:
I guess since this function gets the height dynamically the right thing to do would be to assume it could change some day.

dcrousso reviewed (approved):
nice catch :)

patrickangle committed:
Web Inspector: Screenshots timeline overview records do not adjust their position when zooming in/out
https://bugs.webkit.org/show_bug.cgi?id=242341
rdar://96325691

Reviewed by Devin Rousso.

The position of the images should not only be set during initialization of the record's image element, but also during
each layout.

* Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineOverviewGraph.js:
(WI.ScreenshotsTimelineOverviewGraph.prototype.layout):

Canonical link: https://commits.webkit.org/252148@main
